README.md

```diff
@@ -118,6 +118,28 @@ To create a production build:
 npm run build
 ```
 
+## Creating a Static Production Build
+
+To generate a static production build of your Next.js application, follow these steps:
```

Comment:
We'd also need to move `export const SHELL_COMMAND_URL = 'http://localhost:8080/shell/exec';` as part of `.env` for this, otherwise won't be configurable.